src/main/java/com/google/edith/servlets/ExtractReceipt.java

```diff
@@ -0,0 +1,77 @@
+package com.google.edith.servlets;
+
+import com.google.cloud.documentai.v1beta2.Document;
+import com.google.cloud.documentai.v1beta2.DocumentUnderstandingServiceClient;
+import com.google.cloud.documentai.v1beta2.GcsSource;
+import com.google.cloud.documentai.v1beta2.InputConfig;
+import com.google.cloud.documentai.v1beta2.ProcessDocumentRequest;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ExtractReceipt {
+
+  public static List<Map<String, String>> extractReceipt(String blobKey) throws IOException {
+    String projectId = "edith-step";
+    String location = "us";
+    // For local testing. As blobstore API does not store in GCS in local environment.
+    String inputGcsUri =
+        "gs://edith-receipts/AAANsUmjLAOYQp4Rn9XphEflkVYntq1WQX4m9oczEGqXTn4m7vce4b3d02B0Qe1jYgF2IGJRHTSN6E3u4FSREZrQgbI.SvrR-Q83SYV-TNgy";
```

Comment:
nit: this should be split to more than one line

Reply:
that will be deleted before deploying.